src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -807,7 +825,11 @@ def storeTemplate(self, positionNumber, charBufferNumber = 0x01):
 
         ## DEBUG: Template stored successful
         if ( receivedPacketPayload[0] == FINGERPRINT_OK ):
-            return True
+            ## Return index when positionNumber was -1
+            if ( found == True ):
```

Comment:
This is not the perfect way to go, the method returns different values dependent of internal logic. Better return always the positionNumber or a tuple like (boolean, positionNumber).